app/views/match_contacts/_form.haml

```diff
@@ -1,17 +1,17 @@
 = simple_form_for @match_contacts, url: access_context.match_contacts_path(@match), html: { class: 'current-step-form jContactForm' }, remote: ajax_modal_request? || request.xhr? do |form|
-  = render 'contact_manager/contact_field', contact_type: :shelter_agency_contacts, input_name: 'match_contacts[shelter_agency_contact_ids][]', object: @match_contacts, form: form, current_contact: @current_contact
+  = render 'contact_manager/contact_field', contact_type: :shelter_agency_contacts, input_name: 'match_contacts[shelter_agency_contact_ids][]', object: @match_contacts, match: @match, form: form, current_contact: @current_contact
```

Comment:
Maybe for consistency, we should pass in MatchRoute instead of match?  That would mean it could be the same on the program default contacts side, and the contact_field wouldn't need to see if there was a match, just require a route passed in.